include/brickengine/components/colliders/rectangle_collider_component.hpp

```diff
@@ -5,13 +5,14 @@
 
 class RectangleColliderComponent : public ComponentImpl<RectangleColliderComponent> {
 public:
-    RectangleColliderComponent(double xScale, double yScale, double zScale);
+    RectangleColliderComponent(double xScale, double yScale, double zScale, bool isTrigger);
     static std::string getNameStatic();
 
     // data
     double xScale;
     double yScale;
     double zScale;
+    bool isTrigger;
```

Comment:
Maybe add some documentation as this value name is quite vague if you are not familiar with Unity.